src/components/KTable/KTable.vue

```diff
@@ -479,7 +435,7 @@ const props = defineProps({
    * A prop to pass in an array of headers for the table
    */
   headers: {
-    type: Array,
+    type: Array as PropType<TableHeader[]>,
```

Comment:
I feel like I tried to do this once already but it resulted in needing to explicitly cast a bunch of refs for headers in the consuming apps 🤔 might have been fixed in TS updates since.

Reply:
Yes, but I think it's something we will benefit from? Improves typing in our codebase, even though just slightly

Reply:
I gave up... too much casting and fixing needed during the adoption. Reverting this change back